### ISSUE TITLE
Page up and page down go to beginning/end of the entire document

### ISSUE THREAD

**Issue by mptrsen** [UberWriter/uberwriter#66]
### ENVIROMENT

- Linux distribution: Mint 18
- Desktop Enviroment: XFCE4
- DE version: 4.12.2
- GTK version: 2.24.23

### BUG

Page Up and Page Down go to the beginning and end, respectively, of the document. I would expect them to scroll down a screenful. The usual shortcuts (Ctrl+Home and Ctrl+End) can be used to get to the beginning/end.

**Comment by somas95** [comment]
In my Gnome/Arch they scroll up/down a screenful. Can you please do the same test in gedit? Maybe it's a DE related issue

**Comment by mptrsen** [comment]
Nope, gedit doesn't do that. Uberwriter is the only application that shows this kind of behaviour in my setup.

**Comment by somas95** [comment]
@mptrsen do you still have this issue?
